virtme/commands/run.py

```diff
@@ -1051,7 +1120,7 @@ def do_it() -> int:
         if args.client == "console":
             console_client(args)
         elif args.client == "ssh":
-            ssh_client(args)
+            ssh_client(args, vsock=can_use_ssh_over_vsock(args))
```

Comment:
(detail: why not calling `can_use_ssh_over_vsock` from `ssh_client()`? Or not having vsock as optional?)

Reply:
No good reason, I've moved it to `ssh_client`. Thanks!